src/components/MusicForm/MusicForm.js

```diff
@@ -0,0 +1,86 @@
+import styled from 'styled-components/macro'
+import convertEnergy from '../../services/convertEnergy'
+import { data } from '../tracksShort.json'
+import Button from '../Button'
+import Icon from 'supercons'
+import { NavLink } from 'react-router-dom'
+
+export default function MusicForm({ onCreatePlaylist }) {
+  const energyCategory = Array.from(
```

Comment:
i think u dont need array.from. `new Set` should create a new instance of your array and delete the doubled entries.

Reply:
Hey, without the array.from I got just and object and needed an array. I´ll check again.